nifty-core/src/main/java/com/facebook/nifty/ssl/SslHandlerFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2012-2013 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.nifty.ssl;
+
+import org.jboss.netty.handler.ssl.SslHandler;
+
+/**
+ * A Factory that returns a handler.
+ */
+public interface SslHandlerFactory {
```

Comment:
A similar factory exists already - NiftySecurityFactory - the same one used to plug the Kerberos handlers into the pipeline. Would be nice to reuse that